src/main/scala/Label.scala

```diff
@@ -10,4 +10,7 @@ case class Label(name: String) {
 object Label {
   implicit def stringToLabel(string: String): Label =
     Label(string)
+
+  implicit def labelToString(label: Label): String =
```

Comment:
i dont think we want this. we want to know when we accidentally expect a String where a Label should be provided